### PR TITLE
Felinids don't like being sprayed with water, but aren't degenerates who hate showers

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

This PR is an alternative to https://github.com/tgstation/tgstation/pull/50635.

Being sprayed or splashed by water while you are not on fire as a felinid will give you a -15 moodlet for 10 seconds. Showers will not cause apply this negative moodlet, though, and that is intentional. And yes, this SHOULD work with water from fire extinguishers, if I've read the code for fire extinguishers correctly.

## Why It's Good For The Game

image

## Changelog
:cl: ATHATH
balance: Felinids don't like to be sprayed with water. Showering is fine, though, and so is being sprayed with water while you're on fire.
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
maintainer approval is required

File: code/datums/mood_events/generic_negative_events.dm
Comment (stylemistake) [inline code/datums/mood_events/generic_negative_events.dm:252]: Nice memes, but it probably has to be related to water.

All in all, this seems to be compatible with #50635.

**Review by stylemistake — DISMISSED** [review]

**Comment by spookydonut** [comment]
Feel free to pr this on my branch

**Comment by ATH1909** [comment]
> Feel free to pr this on my branch

It won't cause any merge conflicts with your PR.

Still, I don't think that merging your showering PR (even alongside this one) is a good idea. We should not be punishing players for caring about hygeine, especially when the meme is spraying a cat with a spray bottle, not forcing them to take a shower.

**Comment by Acer202** [comment]
An interaction that people can get behind that isn't silent and unseen that only acts as a bother? Heck yeah. Good stuff, ATH1909. This is what I'm talkin' about. You made it fun.

**Comment by ATH1909** [comment]
> An interaction that people can get behind that isn't silent and unseen that only acts as a bother? Heck yeah. Good stuff, ATH1909. This is what I'm talkin' about. You made it fun.

Thanks.

@spookydonut The description of the sadcat moodlet is now less of a meme.

**Comment by GuillaumePrata** [comment]
Now we just need a mood debuff for being sprayed with water by a felinid you sprayed with water yourself in the past, spray war time.
Memery aside, this was the best of the options presented in my worthless opinion.

**Review by stylemistake — APPROVED** [review]

File: code/modules/reagents/chemistry/reagents/other_reagents.dm
Comment (spookydonut) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:199]: isfelinid() usage like this is why people hate species code

**Comment by HomocidlePancakes** [comment]
So you make felinids dislike being splashed with water and then you make the two most common ways for them to get splashed not affect them. This is silly.

**Comment by Acer202** [comment]
> So you make felinids dislike being splashed with water and then you make the two most common ways for them to get splashed not affect them. This is silly.

He changed it so the method of applying it comes from something that someone else is going to do, not something that you do to yourself. It changed from being an unseen detriment to felinids to more like being an interactive thing that others can enjoy, like laser pointers. The old one would only be funny if you had a sixth sense for felinids taking a shower, while this way lets you partake in shenanigans actively and easily. Punishing players for staying clean / washing off rads is silly. Making it so others interact to cause the effect is the better way, I think.

**Comment by HomocidlePancakes** [comment]
1)
> He changed it so the method of applying it comes from something that someone else is going to do, not something that you do to yourself.

Why is this a bad thing though?

2) Talking about interactivity and fun of a feature as minor as this is not just silly, it’s ridiculous. All it does is make some meter go down that rarely ever affects your char.

**Comment by Acer202** [comment]
> Why is this a bad thing though?

It's was bad because it just acted as an invisible debuff that nobody gets enjoyment from and punishes felinid players for staying clean / rad contamination free to nobody's benefit or enjoyment past the conceptual stage.

> Talking about interactivity and fun of a feature as minor as this is not just silly, it’s ridiculous.
 
Disagree. This will likely encourage people to grab fire extinguishers off of walls to spray at felinids they find distasteful, and the resulting events from that have potential all on their own. Fights, arguments, RP stuff. What was going for it in the old way, do you think? Did you want to drag felinids to the showers? It was an inconvenience that nobody benefited from. This new way has lots of interaction potential.

**Comment by ATH1909** [comment]
Wait, ah shit, showers DO trigger this moodlet still. I guess I'll defer to sticky's PR, since he's incorporated some of my changes (albeit in a harsher manner than I'd have preferred).

**Comment by HomocidlePancakes** [comment]
> This will likely encourage people to grab fire extinguishers off of walls to spray at felinids they find distasteful, and the resulting events from that have potential all on their own.

It only affects your mood, and mood barely affects your char. If you are really that concerned about it go eat a candy from a vendor or get a drink from the bar.

> Fights, arguments, RP stuff. What was going for it in the old way, do you think? Did you want to drag felinids to the showers? It was an inconvenience that nobody benefited from.

I have no idea what you’ve meant to say here. Restate your point.

> This new way has lots of interaction potential.

What? This has much less *interaction* for all I know as the two most common situations this would come up got removed.

> It was bad because it just acted as an invisible debuff that nobody gets enjoyment from and punishes felinid players for staying clean / rad contamination free to nobody's benefit or enjoyment past the conceptual stage.

You can’t be serious, right? It only affects your mood, and mood, again, rarely affects you, and even if does, it can be easily negated. This is very minor, yet you present it as an incredibly serious issue. You either struggling to understand your own points or doing this on purpose.

**Pull request closed without merging**
